Neos.Eel/Classes/Helper/StringHelper.php

```diff
@@ -571,6 +571,22 @@ public function length($string)
     }
 
     /**
+     * Return the count of words for a given string. Remove marks & digits and
+     * flatten all kind of whitespaces (tabs, new lines and multiple spaces)
+     * For example this helper can be utilized to calculate the reading time of an article.
+     *
+     * @param string $unicodeString The input string
+     * @return integer Number of words
+     */
+    public function wordCount($unicodeString)
+    {
+        $unicodeString = preg_replace('/[[:punct:][:digit:]]/', '', $unicodeString);
+        $unicodeString = preg_replace('/[[:space:]]/', ' ', $unicodeString);
+
+        return count(preg_split('/[\n\r\t ]+/', $unicodeString, 0, PREG_SPLIT_NO_EMPTY));
```

Comment:
Why split on `\n\r\t `? You replace `[:space:]` earlier, and the PHP manual says:

> The space characters are HT (9), LF (10), VT (11), FF (12), CR (13), and space (32).